server/__test__/index.test.js

```diff
@@ -229,9 +232,52 @@ describe('authentication', () => {
 
     test('should get news from the news table', async () => {
       const { rows } = await getNews();
-      return expect(rows[5].content).toEqual(
+      const allContent = rows.map((row) => row.content);
+      return expect(allContent).toContain(
         'Researchers simulated a tailgating situation with a small group of overweight but healthy men and examined the impact of eating and drinking on their livers using blood tests and a liver scan.',
       );
     });
+
+    describe('health news', () => {
+      it('router returns 200 after sending the news', async () => {
+        const { status } = await request(app)
+          .get('/api/v1/healthnews/')
+          .expect(200);
+        expect(status).toBe(200);
+      });
+    });
+
+    describe('getFoodCategory query', () => {
+      test('should get Food in Category from the userFoodRelation table', async () => {
+        const { rows } = await getFoodCategory(4, 2);
```

Comment:
It's more readable code we declare agood a variables name for getFoodCategory function and then pass these varibale.